scripts/edit_cfg_cloudlab_hostname.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python0
+import argparse
+import re
+import shutil
+import socket
+import sys
+import os
+
+def backup_file(config_path):
+    backup_path = config_path + '.bak'
+    shutil.copy2(config_path, backup_path)
+    print(f"Backup created at {backup_path}")
+
+def get_hostname_suffix():
+    hostname = socket.gethostname()
+    parts = hostname.split('.', 1)
+    print(parts[1])
+    if len(parts) > 1:
+        return parts[1]
+    else:
+        raise ValueError("Hostname does not contain a suffix part.")
+
+def update_hostnames(config_path, new_suffix):
+    with open(config_path, 'r') as file:
+        config = file.read()
+
+    # Regular expression to find all hostnames prefixed with nodeX
+    pattern = re.compile(r'(hostname\s*=\s*"node\d+\.)[^"]*(")', re.MULTILINE)
+    new_config = pattern.sub(r'\1' + new_suffix + r'\2', config)
+
+    with open(config_path, 'w') as file:
+        file.write(new_config)
+
+    print(f"All hostnames updated to new suffix: {new_suffix}")
+
+def main():
+    if sys.version_info[0] < 3:
+        print("This script requires Python 3.")
```

Comment:
Are you using any package that is only available in Python 3?

Reply:
I use Python3 by default. To avoid potential compatible issues, I add these lines.